mage/installation/source.md

```diff
@@ -24,17 +24,49 @@ Hub](/installation/docker-hub.md) guides.
 
 ## Installing MAGE
 
+### Prerequisits
+
+You need to install few things in order to build MAGE from source. Run following command to set up machine for
+MAGE installation.
+
+```bash
+sudo apt-get update && apt-get install -y \
+    libcurl4        `memgraph` \
+    libpython${PY_VERSION}   `memgraph` \
+    libssl-dev       `memgraph` \
+    openssl         `memgraph` \
+    build-essential `mage-memgraph` \
+    cmake           `mage-memgraph` \
+    curl            `mage-memgraph` \
+    g++             `mage-memgraph` \
+    python3         `mage-memgraph` \
+    python3-pip     `mage-memgraph` \
+    python3-setuptools     `mage-memgraph` \
+    python3-dev     `mage-memgraph` \
+    clang           `mage-memgraph` \
+    git             `mage-memgraph` \
+    --no-install-recommends \
+    && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+```
+
+### Installation process
 **1.** Download the MAGE source code from
 **[GitHub](https://github.com/memgraph/mage)** and run the `setup` script. It
 will generate a `dist` directory with all the needed files:
 
 ```shell
-python3 setup build
+python3 setup build -p /usr/lib/memgraph/query_modules
 ```
 
-:::info
+This command will also copy the contents of the newly created `dist` directory to
+`/usr/lib/memgraph/query_modules` where Memgraph load query modules from. 
+
+**If something doesn't get installed properly, `setup` script will stop installation process. If you have any 
+questions, contact us on [Discord](https://discord.gg/memgraph).**
 
-Make sure you cloned  `mage` github repository with:
+:::warning
+
+Make sure you cloned  `mage` github repository with `--recurse-submodules` flag since it has incorporated Memgraph inside:
```

Comment:
```suggestion
Be sure you cloned the `mage` GitHub repository using the `--recurse-submodules` flag since it has incorporated Memgraph inside:
```

Reply:
since it has incorporated Memgraph inside sounds very strange....

What has what where? :D

Reply:
MAGE has incorporated memgraph haha does this sound okay? What can I change?

Reply:
Make sure you cloned  `mage` github repository with `--recurse-submodules` flag since the image also contains MemgraphDB:

Like this?